pkg/synchronizer/kubernetes/sync_client.go

```diff
@@ -122,7 +122,7 @@ func (sync *KubeSynchronizer) AddTemplates(subType string, hostSub types.Namespa
 
 	select {
 	case sync.tplCh <- rsOrder:
-		klog.V(1).Info("wrote resource request/order to cache")
+		klog.V(1).Infof("wrote resource request/order to cache: %#v", rsOrder)
```

Comment:
A minor comment,

`rsOrder` might be a giant object... print the whole object out could be too much, maybe we should add an `String()` func to `rsOrder`.

In this way, we can do
`klog.V(1).Infof("wrote resource request/order to cache: %s", rsOrder)`

Then within the string func, we can decide what would be print out for debugging or tracing

Reply:
yes, let me revert this change